### DISCUSSION TITLE
vite模式下报错：Uncaught TypeError: Cannot read properties of undefined (reading 'ToolItem')

### DISCUSSION THREAD

**Discussion by anyone0034** [antvis/X6#2564] [Q&A]
无论我用x6还是xflow，只要在vite模式下，就会报错：vite模式下报错：Uncaught TypeError: Cannot read properties of undefined (reading 'ToolItem')，只有我关闭了，才会正常。我想知道vite模式下，如何配置：我现在的配置如下

![微信截图_20220824114541](https://user-images.githubusercontent.com/15609924/186324236-ca2ce7b6-a43e-4761-9c37-7e74c532034a.png)

**Comment by lloydzhou** [comment]
参考这个issue https://github.com/antvis/X6/issues/1181
